typespec-tests/src/main/java/com/resiliency/servicedriven/ResiliencyServiceDrivenAsyncClient.java

```diff
@@ -209,6 +214,10 @@ public Mono<Void> fromNone() {
     public Mono<Void> fromOneRequired(String parameter, String newParameter) {
         // Generated convenience method for fromOneRequiredWithResponse
         RequestOptions requestOptions = new RequestOptions();
+        if (!Arrays.asList("v2").contains(serviceClient.getServiceVersion().getVersion())) {
```

Comment:
It is in generated code. It would have lots of different case in complex one, e.g. some would be "v2", "v3", some "v3", etc. Not easy to "extract common" on generated code (else most method generated here do have some duplicate code).